sys/crypto/modes/ocb.c

```diff
@@ -0,0 +1,328 @@
+/*
+ * Copyright (C) 2018 Mathias Tausig
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_crypto
+ * @{
+ *
+ * @file        ocb.c
```

Comment:
alignment here ;)

Reply:
What do you mean exactly? The file was processed with uncrustify, the alignment is as far as I can tell the same as in the other source files I checked.

Reply:
For some reason Uncrustify didn't align `ocb.c` with the next block (`Offset Codebook...`)

Reply:
Sorry, I still don't get what you mean. The block look perfectly aligend to me. 
The second column in line 13 ("ocb.c") starts at the very same position as the second column in the next line ("Offset Codebook (OCB3) AEAD mode as specified in RFC 7253")

Reply:
It does not matter. We don't name the file name in the doxygen `@file ` special command. Use only `@file` with no argument.

Reply:
OK? 
4933bb7

Reply:
@mtausig sorry, the web browser pulled a trick on me (even with raw mode!)